spec/dummy/app/admin/dashboard.rb

```diff
@@ -0,0 +1,32 @@
+ActiveAdmin.register_page 'Dashboard' do
+  menu priority: 1, label: proc { I18n.t('active_admin.dashboard') }
+
+  content title: proc { I18n.t('active_admin.dashboard') } do
+    div class: 'blank_slate_container', id: 'dashboard_default_message' do
+      span class: 'blank_slate' do
+        span I18n.t('active_admin.dashboard_welcome.welcome')
+        small I18n.t('active_admin.dashboard_welcome.call_to_action')
+      end
+    end
+
+    # Here is an example of a simple dashboard with columns and panels.
```

Comment:
Remove comments

Reply:
We probably should consider removing it from the rails_api_base as well, WDYT?
https://github.com/rootstrap/rails_api_base/blob/master/app/admin/dashboard.rb